LetsBora/LetsBora/Views/ProfileEditViewController.swift

```diff
@@ -0,0 +1,125 @@
+//  LetsBora
+//
+//  Created by Davi  on 21/03/25.
+//
+
+import UIKit
+
+class ProfileEditViewController: UIViewController {
+    // MARK: - UI Components
+    
+    private lazy var nameLabel: UILabel = createLabel(withText: "Nome ")
+    private lazy var nameTextField: UITextField = createTextField()
+    private lazy var nameStackView: UIStackView = createStackView(subViews: [nameLabel,nameTextField])
+    
+    private lazy var emailLabel: UILabel = createLabel(withText: "Email")
+    private lazy var emailTextField: UITextField = createTextField()
+    private lazy var emailStackView: UIStackView = createStackView(subViews: [emailLabel,emailTextField])
+    
+    private lazy var newPasswordLabel: UILabel = createLabel(withText: "Nova Senha")
+    private lazy var newPasswordTextField: UITextField = createTextField()
+    private lazy var newPasswordStackView: UIStackView = createStackView(subViews: [newPasswordLabel, newPasswordTextField])
+    
+    private lazy var confirmPwdLabel: UILabel = createLabel(withText: "Confirmar Senha")
+    private lazy var confirmPwdTextField: UITextField = createTextField()
+    private lazy var confirmPwdStackView: UIStackView = createStackView(subViews: [confirmPwdLabel,confirmPwdTextField])
+    
+    private lazy var saveButton: UIButton = createButton(withTitle: "Salvar")
+    
+    lazy var stackView: UIStackView = createStackView(
+        subViews: [ nameStackView,
+                    emailStackView,
+                    newPasswordStackView,
+                    confirmPwdStackView,
+                    saveButton],
+        spacing: 16)
+    
+    
+    
+    // MARK: - LifeCycle
+    
+    /// Called after the view controller's view is loaded into memory
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Set light gray background color
+        view.backgroundColor = UIColor(red: 242/255, green: 242/255, blue: 247/255, alpha: 1.0)
+        setupView()
+    }
+    
+    // MARK: - Setup View
+    
+    /// Sets up the view hierarchy and constraints
+    private func setupView(){
+        setHierarchy()
+        setConstraints()
+    }
+    
+    /// Adds subviews to the view hierarchy
+    private func setHierarchy(){
+        view.addSubview(stackView)
+    }
+    
+    /// Sets up Auto Layout constraints for the stack view
+    private func setConstraints(){
+        let constraints: [NSLayoutConstraint] = [
+            // Position stack view at the top of safe area with 16pt horizontal margins
+            stackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            stackView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor,constant: 16),
+            stackView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor,constant: -16),
+        ]
+        NSLayoutConstraint.activate(constraints)
+    }
+    
+    // MARK: - Factory Components
+    /// Function to factory labels with diferent texts
```

Comment:
Só um errinho de escrita pra consertar

```suggestion
    /// Function to factory labels with different texts
```